sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -195,12 +196,33 @@ class AstBuilder extends DataTypeAstBuilder
       case _ =>
     }
 
-    val labelText = beginLabelCtx.
+    beginLabelCtx.
       map(_.multipartIdentifier().getText).getOrElse(java.util.UUID.randomUUID.toString).
```

Comment:
nit: for a bit cleaner code, let's move `getOrElse` to the new line and also let's start lines with . instead of ending them with .
this makes it clear immediately that this is a function call extending from the previous line